ext/opentelemetry-ext-azure-monitor/src/opentelemetry/ext/azure_monitor/transport.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+class TransportMixin:
+    def _transmit(self, envelopes):
+        try:
+            response = requests.post(
+                url=self.options.endpoint,
+                data=json.dumps(envelopes),
+                headers={
+                    "Accept": "application/json",
+                    "Content-Type": "application/json; charset=utf-8",
+                },
+                timeout=self.options.timeout,
+            )
+        except requests.RequestException:
+            logger.exception("Transient client side error.")
+            return self.export_result_type.FAILED_RETRYABLE
+
+        text = "N/A"
+        data = None  # noqa pylint: disable=unused-variable
+        try:
+            text = response.text
+        except Exception:  # noqa pylint: disable=broad-except
+            logger.exception("Error while reading response body %s.")
```

Comment:
The format string is wrong.
Why do we want `logger.exception` here?

Reply:
Does it not make sense to log at an `exception` level when an exception is raised?

Reply:
I don't think so. We choose log level based on what we _expect the users to see_.

Reply:
To clarify, is it because we do not expect users to configure their logging levels to expose `exception` level logs?